MDPObjectState[] object_states
simulator_state/SimpleRelation[] relations
